tools/cldr-code/src/main/java/org/unicode/cldr/tool/ExtractMessages.java

```diff
@@ -205,10 +205,11 @@ public void handlePathValue(String path, String value) {
         }
     }
 
-    public static Collator col = Collator.getInstance(ULocale.ENGLISH);
+    public static Collator col = Collator.getInstance(ULocale.ROOT); // freeze below
 
     static {
         col.setStrength(Collator.SECONDARY);
+        col = col.freeze();
```

Comment:
This could also be done with a static centralized collator.

Reply:
Looks like I simply missed that one. I'll do a follow-up PR.